documentation/sphinx/source/administration.rst

```diff
@@ -217,6 +217,10 @@ To temporarily or permanently remove one or more machines from a FoundationDB cl
 
 .. note:: Addresses have the form ``IP``:``PORT``. This form is used even if TLS is enabled.
 
+.. warning:: Localtiy based exclusions should only be used in FDB versions 7.1.42+, 7.3.26+ or newer. Older versions have limited support for locality based exclsuions and therefore the usage of locality based exclusions on those older version is not recommended.
+
+.. warning:: There is currently one known limitation with the locality based exclusions: If a process is serving as a log and the process is currently not reporting to the cluster, e.g. because of a partition, the process will not be detected by the exclude command. The exclusion is still initiated in the background but the exclude command itself will not block until all mutations of this log process are persisted.
```

Comment:
I don't understand `will not block until all mutations of this log process are persisted.`. Does it mean it will block later? Maybe remove "until..."?

Reply:
I removed the second part of the sentence. My intention was to express that the exclude command will not be blocking until the targeted log process is not serving the log role anymore.

Reply:
You mean the mutations/data on this log process won't be moved to another log process or the movement happens but the exclude won't wait until it completes?

Reply:
The latter, the movement is happening but the exclude command will not wait.